consumer/dragonboard410c/guides/aosp.md

```diff
@@ -15,69 +15,31 @@ The following instructions can be used to attempt a build based on the AOSP mast
 
 In this page, we assume that `<ANDROID_TOP>` is the top level folder for your Android environment. e.g.:
 ```shell
-mkdir <ANDROID_TOP>
-cd <ANDROID_TOP>
+$ mkdir <ANDROID_TOP>
+$ cd <ANDROID_TOP>
 ```
 Then run:
 ```shell
-repo init -u https://android.googlesource.com/platform/manifest -b master
-cd .repo
-git clone https://github.com/robherring/android_manifest.git -b master local_manifests
-cd ..
-repo sync -j24
+$ repo init -u https://android.googlesource.com/platform/manifest -b master
+$ git clone https://android-git.linaro.org/git/platform/manifest.git .repo/local_manifests -b dragonboard
+$ repo sync -jX -c # X being the number of logical cores in you build system
 ```
 It might take quite a bit of time to fetch the entire AOSP source code!
 
-# Firmware files (Optional)
-
-Download the firmware setup package from here: https://drive.google.com/open?id=0B2zT38Egh-1TVkZoYUliOXdxaUk, and run the following command to prepare the firmware files in your Android environment:
-```shell
-tar -xf qcom-db410c-NOU.tgz
-./extract-qcom-db410c.sh
-```
-When needed, hit yes, to agree to license.
-
-Note that using this script is a temporary solution until the proper firmware package is published on 96boards website.
-
 # Building AOSP
 ```shell
-source build/envsetup.sh
-lunch linaro_arm64-userdebug
-make -j24
-```
-
-# Building Kernel
```

Comment:
i think we need to keep a section about kernel. while the default build will use pre-built, we want to tell users how to recompile their own kernel. is that something you can add?

Reply:
can you send/link me the instructions for building aosp kernel?

Reply:
can you please check with Amit P.?